renovate-presets/default.json

```diff
@@ -3,22 +3,21 @@
   "description": "Default preset for use with Jellyfin's repos",
   "extends": [
     "config:base",
-    "helpers:pinGitHubActionDigests",
-    ":rebaseStalePrs",
+    ":disableDependencyDashboard",
+    ":enableVulnerabilityAlertsWithLabel(security)",
+    ":prHourlyLimit2",
+    ":prConcurrentLimit20",
```

Comment:
I would remove this. A max of 4 PRs are possible with this config, which I think is really manageable:

* major updates
* minor and patch updates
* CI minor updates
* CI major updates

The fact that Renovate is able to track updates as soon as they're published in the registry is one of the biggest advantages it has in my opinion. Limiting them to 2 per hour defeats that purpose and might make us less resilient to 0-day vulnerabilities that dependencies might have.

Anything will improve the current situation that we have with Dependabot, renovate might also create and spam quite a few PRs, but it's way different in that regard. That's why I would remove this.

Reply:
realistically, even if we would get a PR with a fix for an eventual 0-day the moment it gets published to a repository, would we even be able to publish a release within the next 2 hours ourself?
I don't fell strongly about it, and can remove the hourly limit and will do so shortly :)